animation_nodes/nodes/falloff/mesh_falloff.pyx

```diff
@@ -0,0 +1,86 @@
+import bpy
+import cython
+from ... utils cimport bvh
+from mathutils import Vector
+from ... math cimport Vector3
+from ... base_types import AnimationNode
+from ... data_structures cimport BaseFalloff
+from . constant_falloff import ConstantFalloff
+
+class MeshFalloffNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_MeshFalloffNode"
+    bl_label = "Mesh Falloff"
+    errorHandlingType = "EXCEPTION"
+
+    def create(self):
+        self.newInput("BVHTree", "BVHTree", "bvhTree")
+        self.newInput("Float", "Size", "size")
+        self.newInput("Float", "Falloff Width", "falloffWidth", value = 1)
+        self.newInput("Boolean", "Use Surface", "useSurface", value = True)
+        self.newInput("Boolean", "Use Volume", "useVolume", value = False)
+        self.newInput("Boolean", "Invert", "invert", value = False)
+        self.newInput("Float", "Max Distance", "bvhMaxDistance", minValue = 0, value = 1e6, hide = True)
```

Comment:
Maybe this should be removed and implicitly set to a sufficiently large number.

Reply:
Is `1e10` fine?

Reply:
Yes.